### DISCUSSION TITLE
Welcome to Vritant Discussions!

### DISCUSSION THREAD

**Discussion by bhattabhi013** [bhattabhi013/Vritant#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dakshdeepHERE** [comment]
Hey @bhattabhi013 I was thinking of adding more signup and sign in option in our app. Should we do it?

**Reply by dakshdeepHERE** [reply]
Also when I'm compiling the code in my android emulator it doesn't seem to load any login screen. Is it not there or its a issue from my side @bhattabhi013

**Reply by bhattabhi013** [reply]
Hi @dakshdeepHERE ,
We have removed it for the time being.
